### ISSUE TITLE
Feature requests? Default template,  image display & mobile UI

### ISSUE THREAD

**Issue by dannongruver** [PhiTux/DailyTxT#15]
First of all this is the type of journal I've been searching for. Thank you for creating this! Most other self hosted is "note taking" so it was nice to find such a good representation of real journal app. 
 
I'm wondering about upcoming features and what's the proper way to request features? 

I have requests b/c I like the app:
1) Allow selecting a template as "default" so that it's automatically selected for new days.
2) Display the images that have been uploaded.
3) Mobile UI: There's such limited space on mobile that the focus should be on writing and viewing images. To help  move the "SELECT A TEMPLATE" to the very bottom, remove the green tabs with todays date and last edited, move today's date to the blue menu area at the top and remove the trashcan delete to very bottom with "SELECT A TEMPLATE", move the upload image to blue bar at top and make it a "camera" icon

**Comment by PhiTux** [comment]
Thank you for the feedback :)

That's the correct way (at least for me) to request features.

First of all: I've now got an important other project that has priority and will go on for several months. During this time, there won't be coming any new features to dailytxt (probably). Sorry for that - but I'll still accept pull requests if there are any...

Now to your requests:
1. This feature could perhaps get implemented, yes.
2a. Do you mean: display all images that have ever been uploaded? I don't think that this feature will come. First: because dailytxt isn't primarily an image uploader/viewer. And second: because (the storage solution in combination with) the encryption would make it really slow and produce hard server workload until all the images are decrypted and viewable.
2b. Do you mean: display all images of one day? Whenever you click an uploaded file and this file is an image, then there is a preview popup coming. This behaviour will propably not change to a direct preview of all images of a single day. First: Because there is hardly any space left in the file-upload-area. There's just space left for some mini-preview-icons... And second: same argument as 2a -> decryption of images produces hard server workload and hence should be used with 'caution'.
3. At least on my phone everything is well placed and the writing area is large enough. Yes, the files-area is 'far away' down. But I do not want to produce too large changes just to get the files more to the top... So I don't think that I will change this.

**Comment by dannongruver** [comment]
I completely understand that this isn’t and shouldn’t be your priority (at least not until you start a patreon account or something 😀). So thank you for taking the time to even reply back to these “nice 2 have” features.

1) 👍
2) 2b, thinking more of a thumbnail carousel for daily images.
3) I may have a smaller form factor (iPhone SE). This is really nice to have to simplify the UI by compacting it. Not needed but other journal apps are very focused (but they also have teams of devs supporting)

**Comment by PhiTux** [comment]
Well it would also be nice to have a second developer to support me (at least pull requests are always welcome) 😅
But thanks for the idea with patreon, I did not yet think about that. I just doubt that this would really lead to enough support by the community?! I've no experience with patreon how good it actually works...
